fe/fe-core/src/main/java/com/starrocks/server/NodeMgr.java

```diff
@@ -1080,6 +1086,37 @@ public void setLeader(LeaderInfo info) {
         leaderChangeListeners.values().forEach(listener -> listener.accept(info));
     }
 
+    public List<QueryStatisticsInfo> getQueryStatisticsInfoFromOtherFEs() {
+        List<QueryStatisticsInfo> statisticsItems = Lists.newArrayList();
+        TGetQueryStatisticsRequest request = new TGetQueryStatisticsRequest();
+
+        List<Frontend> allFrontends = getAllFrontends();
+        for (Frontend fe : allFrontends) {
+            if (fe.getHost().equals(getSelfNode().first)) {
+                continue;
+            }
+
+            try {
+                TGetQueryStatisticsResponse response = FrontendServiceProxy
+                        .call(new TNetworkAddress(fe.getHost(), fe.getRpcPort()),
+                                Config.thrift_rpc_timeout_ms,
+                                Config.thrift_rpc_retry_times,
+                                client -> client.getQueryStatistics(request));
+                if (response.getStatus().getStatus_code() != TStatusCode.OK) {
+                    LOG.warn("getQueryStatisticsInfo to remote fe: {} failed", fe.getHost());
```

Comment:
Should we return a message  to tell clients that failed obtain query info in some FE nodes.

Reply:
Not necessary, I think we should still be able to get the query infos from other running FEs